server/src/core/system-lock/tests/system-lock.test.ts

```diff
@@ -16,91 +16,109 @@ describe('SystemLock tests', () => {
     await dbUtils.tearDown();
   });
 
-  beforeEach(async () => {
-    await dbUtils.resetCollectionWith(systemLocks);
-  });
+  // beforeEach(async () => {
+  //   await dbUtils.resetCollectionWith(systemLocks);
+  // }, 10000);
 
   describe('lock', () => {
+    beforeAll(async () => {
+      await dbUtils.resetCollectionWith(systemLocks);
+    }, 10000);
```

Comment:
10 seconds timeout, isn't that quite a lot?

Reply:
Why did you use `beforeAll` when setting the fixtures? An why did you move it inside the `describe('lock')` tests?